meta-oe/recipes-multimedia/webm/libvpx_1.13.1.bb

```diff
@@ -24,8 +24,6 @@ BUILD_LDFLAGS += "-pthread"
 export CC
 export LD = "${CC}"
 
-VPXTARGET:armv5te = "armv5te-linux-gcc"
-VPXTARGET:armv6 = "armv6-linux-gcc"
```

Comment:
if it doesn't build anymore, can you add COMPATIBLE_HOST restriction to indicate that?

Reply:
Sorry, wasn't completely precise. The recipe fails to build for these targets in case the `VPXTARGET` variable is set to the specific toolchain name. In case the fallthrough `generic-gnu` value is used (like it is used for the other arch's), the recipe still builds.

Reply:
OK, then please update the commit message in the commit itself which currently has only the summary (not PR description) to make it more clear.

Reply:
Sure, done.